### DISCUSSION TITLE
Usages and expectations survey (post v1.0.0)

### DISCUSSION THREAD

**Discussion by bpetit** [hubblo-org/scaphandre#372] [General]
Hello, dear contributors and users,

Scaphandre is getting attention and we are structuring the team at Hubblo to improve the project.

In this regard and to guide the project properly, we'd like to know more about your usages and expectations about it.

Could you help us and answer this brief survey? It takes only 5 minutes: [https://framaforms.org/scaphandre-usages-and-expectations-1708965173](https://framaforms.org/scaphandre-usages-and-expectations-1708965173)

Thanks a lot !

Benoit Petit, Scaphandre maintainer

PS: If you didn't see it, we launched [Scaphandre V1](https://hubblo.org/blog/scaphandre-v1-0-is-here/) and a [sponsorship program](https://github.com/sponsors/hubblo-org).

**Comment by TheElectronWill** [comment]
Hello Benoît,

To be fully transparent, Atos and the LIG are working with me to create a new software measurement tool in Rust. We already have support for x86 CPUs, Nvidia GPUs and Jetsons. We have research results that guide us towards a new direction, more flexible and more efficient (in terms of overhead and frequency). I can't share the details right now but the tool will be open-sourced soon :)
I'll provide more information at the Green Days 2024, will you be present?
